### PR TITLE
Add version: Microsoft.PowerToys version V0.82.1

### PR DESCRIPTION
**Pull request by consvc**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/162724)

### CODE REVIEW

File: manifests/m/Microsoft/PowerToys/V0.82.1/Microsoft.PowerToys.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.PowerToys
+PackageVersion: V0.82.1
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerToys/V0.82.1/Microsoft.PowerToys.installer.yaml:5]:
```suggestion
PackageVersion: 0.82.1
```

File: manifests/m/Microsoft/PowerToys/V0.82.1/Microsoft.PowerToys.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.PowerToys
+PackageVersion: V0.82.1
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerToys/V0.82.1/Microsoft.PowerToys.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.82.1
```

File: manifests/m/Microsoft/PowerToys/V0.82.1/Microsoft.PowerToys.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.PowerToys
+PackageVersion: V0.82.1
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerToys/V0.82.1/Microsoft.PowerToys.yaml:5]:
```suggestion
PackageVersion: 0.82.1
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Incorrect package version. After making the changes in the manifest, the version folder would also require renaming to match the new PackageVersion. (i.e., the directory `manifests/m/Microsoft/PowerToys/V0.82.1` needs to be renamed to -> `0.82.1`)

**Comment by mdanish-kh** [comment]
cc @denelon @crutkas

The incorrect version is due to the [release tag](https://github.com/microsoft/PowerToys/releases/tag/V0.82.1) having a capital `V` (`V0.82.1`) instead of the usual lowercase `v` (e.g. `v0.82.0`) in this release of PowerToys. The [wingetcreate CI used to generate the package manifest](https://github.com/microsoft/PowerToys/blob/c8cf9af3c7aeb32b5146d7ed2f017b630431d56c/.github/workflows/package-submissions.yml#L27) strips out a lowercase `v` when selecting the PacakgeVersion from GitHub release tag, but as a capital V was used here, an incorrect PackageVersion came through

**Comment by crutkas** [comment]
This needs to be closed and i'm going to tweak it quickly the folder as it also is incorrect

**Comment by crutkas** [comment]
good catch @mdanish-kh

**Comment by crutkas** [comment]
https://github.com/microsoft/winget-pkgs/pull/162735 has the fixed PR

**Comment by mdanish-kh** [comment]
Thanks! We can close this PR in that case

Close with reason: Replaced by PR #162735;

**Comment by mdanish-kh** [comment]
Lol, seeing this for the first time 

Close with reason: Please bot;

**Pull request closed without merging**